modules/src/ics26_routing/msgs.rs

```diff
@@ -10,7 +10,7 @@ use crate::ics04_channel::msgs::{
     acknowledgement, chan_close_confirm, chan_close_init, chan_open_ack, chan_open_confirm,
     chan_open_init, chan_open_try, recv_packet, timeout, timeout_on_close, ChannelMsg, PacketMsg,
 };
-use crate::ics26_routing::error::{Error, Kind};
+use crate::ics26_routing::error::{self, Error};
```

Comment:
That's a good point. I can modify flex-error to generate the constructor functions inside an `impl Error {}` block. Should be straightforward other than the mass replace.

Reply:
Maybe we can also then drop the `_error` suffix for the constructor?

Reply:
Sure, let's try that out.

Reply:
Hmm unfortunately that is a bit tricky as `Error` is defined as a type alias to `ErrorReport<ErrorDetail, Tracer>`, so we can't define an `impl` block for it. The next best thing we can do is define them as `ErrorDetail::constructor()` instead  of `error::constructor_error()`. What do you think?

Reply:
Sounds good to me!

Reply:
I have updated flex-error to generate the main error type as a struct instead of alias to `ErrorReport`. So now we can use the `Error::constructor()` convention. Right now I am pinning the version of flex-error to the git commit directly, will release a new version of flex-error later on after updating the documentation.